eng/pipelines/aggregate-reports.yml

```diff
@@ -2,14 +2,22 @@ trigger: none
 pr: none
 
 jobs:
-  - job: Generate
+  - job: GenerateReports
+
     variables:
       - template: templates/variables/globals.yml
+
     pool:
       vmImage: 'windows-2019'
+
     steps:
+      - task: UseDotNet@2
+        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+        inputs:
+          version: "$(DotNetCoreSDKVersion)"
+
       - pwsh: mkdir $(System.ArtifactsDirectory)/BuildArtifacts
```

Comment:
Should quote the path like you do below. Unlikely to need it, but coudn't hurt. Also, I recommend single quotes since the variables you have are replaced by AzDO and not PowerShell variables. This makes sure any powershell variables snuck into the value aren't processed unexpectedly.